geoprocessing/test/planning-units-processor.e2e-spec.ts

```diff
@@ -9,28 +9,32 @@ function delay(ms: number) {
     return setTimeout(resolve, ms);
   });
 }
+
 /**
  * @TODO
  * we need to add a couple of test that cath errors on invalid user input.
  */
 describe('planning units jobs (e2e)', () => {
-  jest.setTimeout(5 * 1000);
-  it('executes the child job processor with mock data', async () => {
-    const createPlanningUnitsDTO: Pick<
-      Job<PlanningUnitsJob>,
-      'data' | 'id' | 'name'
-    > = {
-      id: '1',
-      name: 'create-regular-pu',
-      data: E2E_CONFIG.planningUnits.creationJob.valid.customArea({
-        countryCode: 'NAM',
-      }),
-    };
-    const value = await createPlanningUnitGridFromJobSpec(
-      createPlanningUnitsDTO,
-    );
+  it(
+    'executes the child job processor with mock data',
+    async () => {
+      const createPlanningUnitsDTO: Pick<
+        Job<PlanningUnitsJob>,
+        'data' | 'id' | 'name'
+      > = {
+        id: '1',
+        name: 'create-regular-pu',
+        data: E2E_CONFIG.planningUnits.creationJob.valid.customArea({
+          countryCode: 'NAM',
+        }),
+      };
+      const value = await createPlanningUnitGridFromJobSpec(
+        createPlanningUnitsDTO,
+      );
 
-    expect(value).toBeDefined();
-    await Promise.all([delay(4 * 1000)]);
-  });
+      // TODO do actual verification & cleanup (table: planning_units_geom) after test
+      expect(value).toEqual([]);
+    },
+    10 * 1000,
```

Comment:
this may still be just enough, and not always/in every instance - i'd up it even more

Reply:
🤷 hope 5x the above is enough ;)

fixed in https://github.com/Vizzuality/marxan-cloud/pull/184/commits/572b9b0f92976bc9fa554208b6be8ab64e62523a